projects/output-and-error-handling/main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+)
+
+
+func weatherErrorHandler() (string, error){
```

Comment:
There is a lot of inconsistent formatting in this file - can you run your file through `gofmt`? You probably also want to configure your IDE to auto-`gofmt` when you save.